java-checks/src/test/files/checks/IgnoredTestsCheck.java

```diff
@@ -12,4 +12,28 @@ void qix() {}
 
   @org.junit.Ignore("withComment") // compliant : explicit comment about why this test is ignored.
   void foo2() {}
+
+  void assume1() {
+    Assume.assumeTrue(false); // Noncompliant
+  }
+  void assume2() {
+    Assume.assumeFalse(true); // Noncompliant
+  }
+  void assume3() {
+    if(something) {
+      Assume.assumeFalse(true); // compliant, we only detect basic cases
```

Comment:
Why not subscribing to all METHOD_INVOCATION to cover all cases? Is there a valid use case where a logic containing "assumeFalse(true)" makes sense?

Reply:
Being conservative here : I want to highlight "simple case" and not raise issue if there is convoluted logic leading to not executing the test.